app/components/project-category-item.js

```diff
@@ -1,13 +1,59 @@
 import Ember from 'ember';
 
+/**
+  `project-category-item` represents a category with a project. When selected
+  the user is taken to a list of projects using this category.
+
+  ## Default usage
+  ```Handlebars
+  {{project-category-item category=category}}
+  ```
+
+  @class project-category-item
+  @module Components
+  @extends Ember.Component
+ */
 export default Ember.Component.extend({
   tagName: ['li'],
 
+  /**
+    Returns true if 'userCategory' is not empty
+
+    @property selected
+    @type Boolean
+   */
   selected: Ember.computed.notEmpty('userCategory'),
+
+  /**
+    A service that returns all userCategories.
+
+    @property userCategories
+    @type Ember.Service
+   */
   userCategories: Ember.inject.service(),
+
+  /**
+    The current user.
+
+    @property user
+    @type Ember.Object
```

Comment:
`DS.Model` would be more descriptive. I think it would be good to note models that way to differentiate them more.

We should probably go with the ember way. If ember is more specific, we should be too. Maybe a vote on specificity?

Whatever we decide I'll go through all the docs we have and make changes to reflect whatever we standardize here.